package.json

```diff
@@ -130,7 +130,8 @@
     "dependencies": {
         "@deriv/quill-icons": "^1.19.20",
         "@headlessui/react": "^1.7.18",
+        "react-swipeable": "^6.2.1",
         "usehooks-ts": "^3.0.2",
-        "react-swipeable": "^6.2.1"
+        "uuid": "^9.0.1"
```

Comment:
using uuid instead of react-uuid because react-uuid is deprecated (no longer supported/licensed)